src/components/Code/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import cn from 'classnames'
+
+import { htmlToReact } from '../../utils'
+
+import dvc from '../../../config/prismjs/dvc-commands'
+
+const dvcRegex = new RegExp(`dvc (?:${dvc.join('|')})`, 'ig')
+const usageRegex = new RegExp(
+  `(^|\n)\s*(usage|positional arguments|optional arguments)`,
+  'ig'
+)
+const argsRegex = new RegExp(/\-{1,2}[a-zA-Z-]*/, 'ig')
+
+const encodeChars = (rawText: string) => {
+  return rawText.replaceAll('<', '&lt;')
+}
+const pipe = (...args: any[]) => args.reduce((acc, el) => el(acc))
+const linkify = (str: string) => {
+  return `<a class="token args" href='#${str.replaceAll('-', '_')}'>${str}</a>`
+}
+
+const wrapUsage = (text: string) =>
+  text.replace(usageRegex, `<span class="token usage">$&</span>`)
+const wrapDvc = (text: string) =>
+  text.replace(dvcRegex, `<span class="token dvc">$&</span>`)
+const linkifyArgs = (text: string) => text.replace(argsRegex, linkify)
+
+export const wrapWithTags = (text: string) => {
+  text = encodeChars(text)
+  return pipe(text, wrapUsage, wrapDvc, linkifyArgs)
+}
+
+const formatText = (text: string) => {
+  const wrapped = wrapWithTags(text)
+  return htmlToReact(wrapped)
+}
+
+const Code: React.FC<{ className: string }> = ({ className, children }) => {
+  if (className === 'language-usage') {
```

Comment:
Great idea reusing the classes placed by `gatsby-remark-prismjs` to find the code blocks that need linking!

Reply:
Yes, we needed to make sure the existing syntax highlighting worked.